semgrep/semgrep/config_resolver.py

```diff
@@ -463,7 +463,7 @@ def resolve_config(config_str: str) -> Dict[str, YamlTree]:
     else:
         config = load_config_from_local_path(config_str)
     if config:
-        logger.debug(f"loaded {len(config)} configs in {time.time() - start_t}")
+        logger.verbose(f"loaded {len(config)} configs in {time.time() - start_t}")
```

Comment:
if there's time information, that means we can't use --verbose for our pytests and snapshots.